apps/frontend/src/components/review/AssignTechnicalReview.tsx

```diff
@@ -41,7 +41,7 @@ function AssignTechnicalReview({
     technicalReview?.technicalReviewAssigneeId
   );
 
-  const usersData = instrument?.scientists || [];
+  const usersData = instrument?.scientists ? [...instrument?.scientists] : [];

```

Comment:
I guess it might be some typescript issue, but It's a bit weird to see  `?` in the `[...instrument?.scientists]`